apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/LruCacheTest.cs

```diff
@@ -128,7 +128,7 @@ public void MultithreadedUse()
             }
             for (int i = 0; i < threadCount; i++)
             {
-                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5)));
+                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5).Milliseconds));
```

Comment:
I was reacting to the code in the first commit, which had `TimeSpan.FromSeconds(5).Milliseconds`. If it's using `TotalMilliseconds` now, that's fine, at least for test code. (If we need this in multiple places, I'd consider using an extension method...)